resources/views/enable-tfa-via-app.blade.php

```diff
@@ -0,0 +1,54 @@
+
+@extends('layouts.layout')
+
+@section('content')
+
+  <div class="container">
+    <div>
+      <h1>Enable Google Authenticator</h1>
+    </div>
+    <?php if (isset($errorMessage)): ?>
```

Comment:
Please use [blade control structures](https://laravel.com/docs/5.2/blade#control-structures), not plain php blocks opening and closing.